concurrent_test.go

```diff
@@ -0,0 +1,98 @@
+package bbolt_test
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	bolt "go.etcd.io/bbolt"
+	"go.etcd.io/bbolt/internal/btesting"
+)
+
+func TestRepeatableRead(t *testing.T) {
+	const (
+		readInterval  = 50 * time.Millisecond
+		writeInterval = 10 * time.Millisecond
+		testDuration  = 30 * time.Second
+	)
+
+	bucket := []byte("data")
+	key := []byte("foo")
+
+	// prepare the db
+	t.Log("Create the db")
```

Comment:
nit: If we want such log (I have nothing against) it should be part of `btesting.(Must)CreateDB`